src/EA.Weee.Web/Areas/AatfReturn/Views/CheckYourReturn/Index.cshtml

```diff
@@ -256,26 +261,71 @@
     <div class="govuk-!-padding-bottom-6"></div>
 
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-one-quarter">
-            @using (Html.BeginRouteForm(AatfRedirect.Default, FormMethod.Post))
-            {
-                @Html.AntiForgeryToken()
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter">
+                @using (Html.BeginRouteForm(AatfRedirect.Default, FormMethod.Post))
+                {
+                    @Html.AntiForgeryToken()
 
-                @Html.HiddenFor(m => m.ReturnId)
-                @Html.HiddenFor(m => m.OrganisationId)
+                    @Html.HiddenFor(m => m.ReturnId)
+                    @Html.HiddenFor(m => m.OrganisationId)
 
-                <div class="govuk-form-group">
-                    <button class="govuk-button" type="submit">Continue</button>
-                </div>
-            }
+                    <div class="govuk-form-group">
+                        <button class="govuk-button" type="submit">Continue</button>
+                    </div>
+                }
 
+            </div>
         </div>
-        <div class="govuk-grid-column-one-quarter">
-            <div class="govuk-body govuk-!-font-size-19">
-                <a class="govuk-link govuk-link--no-visited-state">
-                    @Html.RouteLink("AATF task list", AatfRedirect.Default, new { returnId = Model.ReturnId, action = "Index", controller = "AatfTaskList" }, null)
-                </a>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter">
+                <div class="govuk-body govuk-!-font-size-19">
+                    <a class="govuk-link govuk-link--no-visited-state">
+                        @Html.RouteLink("AATF task list", AatfRedirect.Default, new { returnId = Model.ReturnId, action = "Index", controller = "AatfTaskList" }, null)
+                    </a>
+                </div>
             </div>
         </div>
-    </div>
-</section>
\ No newline at end of file
+        </div>
+</section>
+
+<script type="text/javascript">
+    document.getElementById('collapsibleHref').onclick = function() {
```

Comment:
Extract this JS out into it's own scripts, add it to a bundle in Web\App_Start\BundleConfig, then render it on this page. Similar to to how it's done on SentOnCreateSiteOperator Index page and the sent-on-operator-address JS script